src/_guides/language/language-tour.md

```diff
@@ -634,6 +634,16 @@ Dart list:
 var list = [1, 2, 3];
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  If you're wondering whether this list literal is
+  [strong mode compliant](/guides/language/sound-dart), it is. Thanks to
+  [type inference](/guides/language/sound-dart#type-inference),
+  the analyzer infers a list of type `int`.
```

Comment:
Darn it, I can't leave well enough alone. I think we need to be more specific about the type. Maybe:

...the analyzer infers that `list` has the type `List<int>`.

(But check that I got the type right!)